### PR TITLE
Update the spec versions and descriptors for Java EE 7

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: cdi-interceptors/src/main/resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -15,10 +15,10 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 -->
-<persistence version="2.0" xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<persistence version="2.0" xmlns="http://xmlns.jcp.org/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (rafabene) [inline cdi-interceptors/src/main/resources/META-INF/persistence.xml]: Shouldn't we use 2.1 here ?

**Comment by rafabene** [comment]
Will you include faces-config.xml, web.xml, application.xml in other commit ?

**Comment by sgilda** [comment]
@rafabene : I must have missed those. Yes, I'll do that in a separate commit. Let me know when I can merge this one and I'll create a new branch for these changes.

File: cdi-interceptors/src/main/resources/META-INF/persistence.xml (hunk 1 above)
Comment (sgilda) [inline cdi-interceptors/src/main/resources/META-INF/persistence.xml]: Yes. Good catch!

**Comment by rafabene** [comment]
It's good to go :) Thanks!

**Pull request merged**